### PR TITLE
Makes  vscode the default editor

### PR DESCRIPTION
**Pull request by MiraMarshall**
_Description of what this PR is changing or adding, and why:_
[Set up editor staging](https://sm-flutter-web.web.app/get-started/editor/)
[Test drive staging](https://sm-flutter-web.web.app/get-started/test-drive/)

This PR makes VS code the default editor on pages with multiple editors. 
_Issues fixed by this PR (if any):_ Fixes #ISSUE-NUMBER
#7531 
Screen Shot 2022-09-08 at 10 28 06 AM

## Presubmit checklist
- [x] This PR doesn’t contain automatically generated corrections (Grammarly or similar).
- [x] This PR follows the [Google Developer Documentation Style Guidelines](https://developers.google.com/style) — for example, it doesn’t use _i.e._ or _e.g._, and it avoids _I_ and _we_ (first person).
- [x] This PR uses [semantic line breaks](https://github.com/dart-lang/site-shared/blob/main/doc/writing-for-dart-and-flutter-websites.md#semantic-line-breaks) of 80 characters or fewer.

### CODE REVIEW

File: src/get-started/editor.md [hunk 1]
```diff
@@ -23,10 +23,10 @@ that's OK, skip ahead to the [next step: Test drive][].
 {% comment %} Nav tabs {% endcomment -%}
 <ul class="nav nav-tabs" id="editor-setup" role="tablist">
   <li class="nav-item">
-    <a class="nav-link active" id="androidstudio-tab" href="#androidstudio" role="tab" aria-controls="androidstudio" aria-selected="true">Android Studio and IntelliJ</a>
+    <a class="nav-link" id="vscode-tab" href="#vscode" role="tab" aria-controls="vscode" aria-selected="true">Visual Studio Code</a>
   </li>
-  <li class="nav-item">
-    <a class="nav-link" id="vscode-tab" href="#vscode" role="tab" aria-controls="vscode" aria-selected="false">Visual Studio Code</a>
+    <li class="nav-item">
```
Comment (khanhnwin) [inline src/get-started/editor.md:28]: unintended whitedpace?

File: src/get-started/editor.md [hunk 2]
```diff
@@ -23,10 +23,10 @@ that's OK, skip ahead to the [next step: Test drive][].
 {% comment %} Nav tabs {% endcomment -%}
 <ul class="nav nav-tabs" id="editor-setup" role="tablist">
   <li class="nav-item">
-    <a class="nav-link active" id="androidstudio-tab" href="#androidstudio" role="tab" aria-controls="androidstudio" aria-selected="true">Android Studio and IntelliJ</a>
+    <a class="nav-link" id="vscode-tab" href="#vscode" role="tab" aria-controls="vscode" aria-selected="true">Visual Studio Code</a>
```
Comment (khanhnwin) [inline src/get-started/editor.md:26]: should this links have the `active` class?

File: src/get-started/test-drive/index.md [hunk 3]
```diff
@@ -19,10 +19,10 @@ Flutter apps.
 {% comment %} Nav tabs {% endcomment -%}
 <ul class="nav nav-tabs" id="editor-setup" role="tablist">
   <li class="nav-item">
-    <a class="nav-link active" id="androidstudio-tab" href="#androidstudio" role="tab" aria-controls="androidstudio" aria-selected="true">Android Studio and IntelliJ</a>
+    <a class="nav-link" id="vscode-tab" href="#vscode" role="tab" aria-controls="vscode" aria-selected="false">Visual Studio Code</a>
   </li>
   <li class="nav-item">
-    <a class="nav-link" id="vscode-tab" href="#vscode" role="tab" aria-controls="vscode" aria-selected="false">Visual Studio Code</a>
+    <a class="nav-link active" id="androidstudio-tab" href="#androidstudio" role="tab" aria-controls="androidstudio" aria-selected="true">Android Studio and IntelliJ</a>
```
Comment (khanhnwin) [inline src/get-started/test-drive/index.md:25]: is the active class supposed to be here?

**Comment by sfshaza2** [comment]
@MiraMarshall, thanks for the image of the new UI, but can you provide the URL? I'd like to look at it in situ.

**Comment by sfshaza2** [comment]
cc @anderdobo

File: src/get-started/editor.md (hunk 2 above)
Comment (MiraMarshall) [reply src/get-started/editor.md]: Nice catch. Yes, it should be active.

**Comment by MiraMarshall** [comment]
> @MiraMarshall, thanks for the image of the new UI, but can you provide the URL? I'd like to look at it in situ.

Sure thing! https://sm-flutter-web.web.app/get-started/test-drive/

**Pull request merged**